packages/neuron-wallet/src/main.ts

```diff
@@ -15,6 +16,7 @@ app.on('ready', async () => {
 
   NetworksService.getInstance().notifyAll()
   WalletService.getInstance().generateAddressesIfNecessary()
+  register()
```

Comment:
`registerListeners` could be more clear.

Reply:
updated